pinot-core/src/test/java/org/apache/pinot/realtime/converter/RealtimeSegmentConverterTest.java

```diff
@@ -46,28 +46,11 @@ public void testNoVirtualColumnsInSchema() {
     String segmentName = "segment1";
     VirtualColumnProviderFactory.addBuiltInVirtualColumnsToSegmentSchema(schema, segmentName);
     Assert.assertEquals(schema.getColumnNames().size(), 5);
-    Assert.assertEquals(schema.getTimeFieldSpec().getIncomingGranularitySpec().getTimeType(), TimeUnit.MILLISECONDS);
 
     RealtimeSegmentConverter converter =
         new RealtimeSegmentConverter(null, "", schema, "testTable", tableConfig, segmentName, "col1");
 
     Schema newSchema = converter.getUpdatedSchema(schema);
     Assert.assertEquals(newSchema.getColumnNames().size(), 2);
-    Assert.assertEquals(newSchema.getTimeFieldSpec().getIncomingGranularitySpec().getTimeType(), TimeUnit.DAYS);
-  }
-
-  @Test
-  public void testNoTimeColumnsInSchema() {
-    Schema schema = new Schema();
```

Comment:
Do we still want this support? (Realtime table without time column)

Reply:
realtime table without time will never happen right?
Also after changing the getUpdatedSchema method, this test becomes irrelevant

Reply:
I think @fx19880617 added that support. Any opinion on this?